sambahsa.md

```diff
@@ -2542,31 +2543,29 @@ SOME, FEW, MANY AND OTHERS
 
 **Many *****(adjective)***** –** mult {mult}
 
-**Too (much/many) *****(adverb)***** –** pior {pyor}
+**Too (much/many) *****(adverb or adjective)***** –** pior {pyor}
 
 **Quite *****(adverb)***** – **destull {dëstUl}
 
 **Several –** plur {plur}
 
-**More *****(adverb) (comparative)***** –** meis {meys}
+**More *****(adverb or adjective) (comparative)***** –** meis {meys}
 
-**Less *****(adverb)***** *****(comparative)***** –** minter {mIntër}
+**Less *****(adverb or adjective)***** *****(comparative)***** –** minter {mIntër}; mins {mins}
 
-**Most *****(adverb) (superlative)***** –** meist {meyst}
+**Most *****(adverb or adjective) (superlative)***** –** meist {meyst}
 
 **Less** ***(adverb)***** *****(superlative)***** **– minst {minst}
 
-**Tanto, Tantos – **tant {tant}
+**So much, so many – **tant {tant}
 
 **Each *****(more than two)***** –** ielg {yElg}
 
 **Each *****(of two)*****, either –** Ieter {yEtër}
 
-**All the –** vasyi {vAzyi} *(masculine)*
+**All the –** vasyi {vAzyi} *(masculine & undetermined nominative plural)*
 
-**All the –** vasya {vAzya} *(feminine)*
-
-**All the –** vasyo {vAzyo} *(neutral)*
+**All the –** vasya {vAzya} *(neutral plural nominative & accusative)*
 
```

Comment:
"vasyas"  (I deleted the singular form because it logically doesn't exist : vasyi is "all the + plural noun"). When you refer to the entirety of a substantive, use the adjective "hol" (like "whole" in English)